### DISCUSSION TITLE
train error

### DISCUSSION THREAD

**Discussion by foreverfreedown** [deepmodeling/deepmd-kit#854] [Q&A]
When my training reached 386,000 steps, the mission was suddenly aborted. The error message was as follows. Could you tell me what caused it
![Uploading 捕获.PNG…]()

**Comment by foreverfreedown** [comment]
![捕获](https://user-images.githubusercontent.com/49577178/125645499-6972f1a4-093b-4ee2-a545-44ba938aa92f.PNG)

**Reply by foreverfreedown** [reply]
This is an error message, the picture above did not upload well.

**Comment by njzjz** [comment]
This should be a problem of your operating system and not related to deepmd-kit.

**Reply by foreverfreedown** [reply]
Thank you for your reply. I think so too.
